File: C++/bubble_sort_v2.cpp
#include<iostream>

using namespace std;

int main() {
	int aux, N = 10;
	int vetor[N];

	for (int i = 0; i < N; i++) {
		cout << "Digite o " << i + 1 << "° número: ";
		cin >> vetor[i];
	}

	for (int i = 0; i < (N - 1); i++) {
		for (int j = (N - 1); j > i; j--) {
			if (vetor[j] < vetor[j - 1]) {
				aux = vetor[j];
				vetor[j] = vetor[j - 1];
				vetor[j - 1] = aux;
			}
		}
	}

	for (int i = 0; i < N; i++) {
		cout << (i + 1) << "° número: " << vetor[i] << "\n";
	}

	return 0;
}

File: python/bubble_sort_v2.py

N = 5
aux = -1
vetor = [None] * N

# vetor = list(range(N))

for i in range(N):
	vetor[i] = int(input('Entre com o ' + str(i + 1) + 'º valor: '))

for i in range(N - 1):
	for j in range((N - 1), 0, -1):
		if j > i:
			if vetor[j] < vetor[j - 1]:
				aux = vetor[j]
				vetor[j] = vetor[j - 1]
				vetor[j - 1] = aux
		else:
			break

for i in range(N):
	print(str(i + 1) + '° número: ' + str(vetor[i]), end='\n')
